src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.star {
+  background: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  border-radius: 0.5px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--5 .star,
```

Comment:
please add proper `:nth-child` because what if you’ll have more than 5 stars children but the number of active stars will be `5`?